about.html

```diff
@@ -32,7 +32,7 @@
       <button class="my-menu-btn" type="button">
         <i class="fas fa-bars"></i>
       </button>
-      <a href="index.html"><img class="my-logo" src="/images/anhembi-logo.svg" alt="logo Anhembi Parque" /></a>
+      <a href="index.html"><img class="my-logo" src="./images/anhembi-logo.svg" alt="logo Anhembi Parque" /></a>
       <nav class="my-nav-menu">
```

Comment:
- [ ] OPTIONAL: I strongly suggest you to check the mobile `navbar` the`x` button is not behaving as expected: 

https://user-images.githubusercontent.com/77038610/152654142-cc1d6c15-4cef-4358-aa1e-ee44310cc49d.mp4